project/accounts/urls.py

```diff
@@ -1,7 +1,10 @@
 from django.urls import path
-from .views import SignUpView, user_profile
+
+# from .views import SignUpView, user_profile
+from .views import SignUpView, UserProfileUpdateView
```

Comment:
We can remove the commented code, since your CBV approach is much nicer :smile: 
```suggestion
from .views import SignUpView, UserProfileUpdateView
```